src/styled/PlanRoom/breakpointsButton.tsx

```diff
@@ -0,0 +1,12 @@
+import { createBreakpoints } from '@chakra-ui/theme-tools';
+
+const breakpoints = createBreakpoints({
```

Comment:
There is no point in defining breakpoints, where each one has the same value.

Reply:
The file will be deleted and the parameter next to the button will be applied